@here/harp-mapview/lib/text/Placement.ts

```diff
@@ -477,6 +500,8 @@ function placePointLabelChoosingAnchor(
     // No placement found - revert back the original alignment.
     label.layoutStyle!.horizontalAlignment = basePlacement.h;
     label.layoutStyle!.verticalAlignment = basePlacement.v;
+    delete label.bounds;
```

Comment:
Afaik this line is not needed, setting the bounds to undefined will suffice. I haven't seen this idiom used in our codebase, at least applied to typescript class members.

Reply:
Done.